src/mercury_engine_data_structures/formats/bmtun.py

```diff
@@ -48,3 +49,15 @@ class Bmtun(BaseResource):
     @functools.lru_cache
     def construct_class(cls, target_game: Game) -> Construct:
         return BMTUN
+
+    def get_tunable(self, tunable: str, param: str) -> Container:
+        classes = self.raw.classes
+        if tunable not in classes:
+            raise ValueError(f"Unknown tunable: {tunable}!")
+        if param not in classes[tunable].tunables:
+            raise ValueError(f"Unknown tunable param: {param}!")
+        return self.raw.classes[tunable].tunables[param]
+
+    def set_tunable(self, tunable_name: str, param: str, value: Any) -> None:
+        tunable = self.get_tunable(tunable_name, param)
+        tunable.value = value
```

Comment:
rather than doing it like this, split off the error checking currently in `get_tunable()` into a `_check_tunable_exists()` function called by both of these, and then do `self.raw.classes[class].tunables[tunable].value = value`